src/libraries/System.Runtime.InteropServices/tests/System.Runtime.InteropServices.UnitTests/System/Runtime/InteropServices/NFloatTests.cs

```diff
@@ -793,7 +793,8 @@ public void IsNaN(float value)
         public void IsNegative(float value)
         {
             bool result = NFloat.IsNegative(value);
-            Assert.Equal(float.IsNegative(value), result);
+            bool expected = Environment.Is64BitProcess ? double.IsNegative(value) : float.IsNegative(value);
```

Comment:
nit: It might be nice if this were setup similarly to other tests that have to do the same.

See `IsNormal` for an example.

Reply:
ok